tests/functional/network_policy/test_network_policy.py

```
@@ -0,0 +1,453 @@
+        logger.info(f"Routes in {namespace}: {route_names}")
+
+        tested = 0
+        for route in routes:
+            name = route["metadata"]["name"]
+            host = route.get("spec", {}).get("host", "")
+            tls = route.get("spec", {}).get("tls")
+            scheme = "https" if tls else "http"
+            url = f"{scheme}://{host}"
+
+            try:
+                resp = requests.get(url, timeout=15, verify=False)
```

Ruff (0.16.0)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)